packages/okta-angular/src/okta/okta.config.ts

```diff
@@ -16,8 +16,9 @@ export interface OktaConfig {
   issuer?: string;
   redirectUri?: string;
   clientId?: string;
-  responseType?: string;
+  responseType?: string | string[];
```

Comment:
I don't think we should allow this. It exposes an implementation detail of auth-js

Reply:
Yes, but necessary?  Someone opened issues because this was a limitation for them.